scripts/core/effect.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from typing import Any, Dict
+
+log = logging.getLogger(__name__)
+
+
+class Effect:
+    @classmethod
+    def from_dict(cls, **parameters):
+        instance = cls(**parameters)
+        return instance
+
+
+# TODO: replace with plugins =================================================
+from scripts.core.effects.add_item import AddItemEffect
+from scripts.core.effects.attribute_modifier import AttributeModifierEffect
+from scripts.core.effects.sildreths_signature import SildrethsSignatureEffect
+
+effect_classes: Dict[str, Any] = dict()
+for klass in (
```

Comment:
`klass`?  Is that just as another version of `class`?

Reply:
klass is a common way of using "class" as a variable name.  another common way is using and underscore, like `class_`